app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -108,7 +108,7 @@ def self.hostname_required?
     false
   end
 
-  def self.default_blacklisted_event_names
+  def self.filtered_event_names
     Settings.ems.ems_amazon.blacklisted_event_names
```

Comment:
Can we add a comment here, similar to vmware, about why we aren't inheriting the base class method?